src/main/java/mate/academy/bookshop/dto/CreateBookRequestDto.java

```diff
@@ -1,15 +1,34 @@
 package mate.academy.bookshop.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotBlank(message = "can not be blank")
```

Comment:
Better to specify field name in error message

Reply:
It will duplicate field name, from which I'll have error